packages/documentation/src/pages/platform/site-structure/application-structure.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Page and application Javascript structure
+tags: page, bundle, react, javascript, structure, webpack, site-wide
+---
+
+# Page and application Javascript structure
+
+In this doc we'll go over how a page on VA.gov is related to the application code located in vets-website.
+
+## Javascript bundles and static pages
+
+If you look at the source for the `/my-va` page when running locally you'll see a group of three script tags:
+
+```html
+<script defer nomodule="" src="/generated/polyfills.entry.js"></script>
+<script defer src="/generated/vendor.entry.js"></script>
+<script defer src="/generated/dashboard.entry.js"></script>
+```
+
+The vendor and polyfills bundles are on every page, but the dashboard bundle is specific to this particular React app. In the source for the `/my-va` page, you'll see this frontmatter:
+
+```md
+---
+title: My VA
+layout: page-react.html
+entryname: dashboard
+---
+```
+
+When this page is built, the `entryname` property tells our build process how to construct the path for the third script tag in the above group. For pages that don't have an `entryname`, the `static-pages` bundle will be used.
+
+In `vets-website`, there will be a manifest.json (or manifest.js) file with a matching entry name. In this case we can search for `entryName: 'dashboard'` and find [src/applications/personalization/dashboard/manifest.js](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/dashboard/manifest.js). The `entryFile` property in the manifest will tell you the entry point for this application, the file that Webpack uses as its entry for creating the `dashboard` bundle.
+
+## Structure of a page
+
+A typical application page has several independent React components. Here's an abbreviated screenshot of the `/my-va` page:
+
+![Abbreviated dashboard with red boxes drawn around React mount points](../../../images/platform/react-mount-points.png)
+
+Each of the areas marked with a red box is a separately mounted React component. These React components shared a common Redux store:
+
+```mermaid
```

Comment:
this didn't render for me locally

Reply:
It seems to break if there's a change to the page, but it renders for me whenever I restart the `develop` process:

![Screen Shot 2019-05-23 at 12 04 57 PM](https://user-images.githubusercontent.com/634932/58268205-fef88180-7d52-11e9-8ccc-a1dc8f20bd24.png)